### DISCUSSION TITLE
xpdf suport PDF without embedded fonts

### DISCUSSION THREAD

**Discussion by ragebear00** [dnGrep/dnGrep#1113] [Show and tell]
the ptdtotext.exe (or PDF plugin) does not support PDF with CJK character but without font embedded.

A sample file 
[2.pdf](https://github.com/dnGrep/dnGrep/files/15307270/2.pdf)

To resolve above, you can copy the in the /plugin/PDFSearch/xpdf folder 
[simplified Chinese.zip](https://github.com/dnGrep/dnGrep/files/15307297/simplified.Chinese.zip)
![Snipaste_2024-05-14_18-54-50](https://github.com/dnGrep/dnGrep/assets/168278723/bb73e8ad-b711-4bdf-ac77-9ddefc2941d1)

dnGREP now support CJK Simplified Chinese, thanks to https://www.cnblogs.com/issacnew/p/17493932.html

**Comment by doug24** [comment]
Or you can look at the [dnGrep PDF help page](https://github.com/dnGrep/dnGrep/wiki/PDF-Search).
